src/main.cpp

```diff
@@ -8,10 +8,13 @@
 #include <thread>
 #include <vector>
 
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include <stb/stb_image_write.h>
+
 #include "render.hpp"
 
 // Function to save image data in ppm format
-void save_image(int width, int height, auto& fb_data) {
+void dump_image_ppm(int width, int height, auto& fb_data) {
```

Comment:
``dump_image_ppm` vs `save_image_png`.

But I suggest we keep only the png version under the name `save_image`